roxctl/deployment/check/check_test.go

```diff
@@ -340,15 +366,15 @@ func (d *deployCheckTestSuite) TestConstruct() {
 
 func (d *deployCheckTestSuite) TestValidate() {
 	cases := map[string]struct {
-		file       string
+		file       []string
```

Comment:
nit: s/file/files

Reply:
Oh, must've missed that one when refactoring. Changing it.